src/components/DataBrowser.vue

```diff
@@ -147,8 +159,9 @@ export default {
         promises.push(this.girderRest.get(GIRDER_ITEM_ENDPOINT, { params: itemParams }));
       }
       const responses = (await Promise.all(promises)).map(response => response.data);
-      const rows = [].concat.apply(...responses).map(item => ({
+      const rows = responses.flat().map(item => ({
```

Comment:
This change is just for readability, but the downside is that `Array.prototype.flat` only exists in node>=11, and is not supported on Edge or IE, and I'm unsure of exactly what our polyfill situation is. I can revert this if it's breaking compatibility with environments that we care about.

Reply:
I actually ran into this very recently.  I used flat() on some of my video component stuff and found that it was only recently supported in chrome (version 69  -- the last version with more angled tabs). The current babel config won't polyfill for this, but I didn't look into what would.